src/Blazored.LocalStorage/Blazored.LocalStorage.csproj

```diff
@@ -30,20 +30,22 @@
     <RazorLangVersion>3.0</RazorLangVersion>
   </PropertyGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'net5.0'">
+  <ItemGroup Condition="'$(TargetFramework)' == 'net5.0' or '$(TargetFramework)' == 'net6.0'">
     <SupportedPlatform Include="browser" />
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.1' OR '$(TargetFramework)' == 'netcoreapp3.1'">
-    <PackageReference Include="Microsoft.AspNetCore.Components" Version="$(DotNet3Version)" />
     <PackageReference Include="Microsoft.AspNetCore.Components.Web" Version="$(DotNet3Version)" />
   </ItemGroup>
   
   <ItemGroup Condition="'$(TargetFramework)' == 'net5.0'">
-    <PackageReference Include="Microsoft.AspNetCore.Components" Version="$(DotNet5Version)" />
```

Comment:
`Microsoft.AspNetCore.Components.Web` has a dependency to this package so no need to extra reference it.